src/PoolManager.sol

```diff
@@ -259,7 +259,8 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
 
     /// @inheritdoc IPoolManager
     function take(Currency currency, address to, uint256 amount) external override noDelegateCall isLocked {
-        _accountDelta(currency, amount.toInt128());
+        // flipping a postive number in an int is always safe
```

Comment:
I find this comment confusing. I take it as saying:" flipping a uint is implicitly safe", but in fact we're using a safeCast library, so it's not actually implicitly safe and we need to make it safe. I think using the library function here is self explanatory and the comment is causing confusion. ??

Reply:
Oh what my comment is trying to refer to is the `-` not the `.toInt128`. Its this scenario:
- you have an int128 `x`, and you know that `x>0`
- its completely safe do just do `int128 y = -x`

whereas if you have an int128 `x` and you know that `x<0`, for this case it is NOT save to do `int128 y = -x`. This is because the lowest possible negative number in an int type, does not have a corresponding positive value.

that is to say `type(int128).min * -1 = type(int128).max + 1` so you cannot safely flip in 1 direction, but you can in the other direction

idk how to put that in a single comment though 😂 i'll try again

Reply:
just went with `subtraction must be safe`